core/src/main/scala/org/bitcoins/core/protocol/dlc/models/ContractDescriptor.scala

```diff
@@ -97,6 +99,22 @@ case class NumericContractDescriptor(
     extends ContractDescriptor
     with TLVSerializable[ContractDescriptorV1TLV] {
 
+  private val maxValue: Long = (Math.pow(2, numDigits) - 1).toLong
+
+  require(outcomeValueFunc.points.head.isEndpoint,
+          "Payout curve must start with an end point")
+  require(
+    outcomeValueFunc.points.head.outcome == 0,
+    s"Payout curve must start with its minimum value, 0, got ${outcomeValueFunc.points.head.outcome}")
+
+  require(outcomeValueFunc.points.last.isEndpoint,
+          "Payout curve must end with an end point")
+
+  require(
+    outcomeValueFunc.points.last.outcome == maxValue,
+    s"Payout curve must end with its maximum value, $maxValue, got ${outcomeValueFunc.points.last.outcome}"
+  )
```

Comment:
```
  require(
    outcomeValueFunc.points.last.outcome == maxValue,
    s"Payout curve must end with its maximum value, $maxValue, got ${outcomeValueFunc.points.last.outcome}. You must define the payout interval from ${outcomeValueFunc.points.last.outcome}-$maxValue"
  )
```

Reply:
Basically I added the the interval the user needs to define to the error message